recipes/mercurial/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "3.9" %}
+
+package:
+  name: mercurial
+  version: {{ version }}
+
+source:
+  fn: mercurial-{{ version }}.tar.gz
+  url: https://mercurial-scm.org/release/mercurial-{{ version }}.tar.gz
+  sha256: 834f25dcff44994198fb8a7ba161a6e24204dbd63c8e6270577e06e6cedbdabc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python <3.0
+  run:
+    - python <3.0
```

Comment:
You can remove that and add a skip statement in the build section above like this:

`skip: True  # [py3k]`